platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/capabilities/CapabilitiesConflictResolutionIntegrationTest.groovy

```diff
@@ -260,4 +257,74 @@ class CapabilitiesConflictResolutionIntegrationTest extends AbstractModuleDepend
             }
         }
     }
+
+    @RequiredFeature(feature = GradleMetadataResolveRunner.REPOSITORY_TYPE, value = "maven")
```

Comment:
Should we move this to `CapabilitiesConflictResolutionIssuesIntegrationTest` and attach an `@Issue` to it?

Reply:
This test was added by a previous PR by louis that did fix an existing issue. 

We just never linked to that.

This question was rhetorical. We should link this regression test to the original issue that caused it to be added, and we should move it to `CapabilitiesConflictResolutionIssuesIntegrationTest`